programs/lfx-mentorship/2024/01-Mar-May/project_ideas.md

```diff
@@ -18,4 +18,15 @@
 
 ## Proposed Project ideas
 
+### Cilium
+
+#### Governance Documentation
+
+- Description: This project will focus on governance documentation for the Cilium project with two key parts. First, the governenace documentation should be moved out of the main docs and into the community repo. Second, we need to do an inventory of all of the repos under the project and come up with a lifecycle for them.
+- Expected Outcome: Governance docs in community repo. All repos accounted for and with a lifecycle plan.
```

Comment:
While I love having this kind of work part of a mentoring program, do you think that there are more tasks?

I obviously don't know everything about that's asked in the upstream issues and I might be underestimating, but to me, the project proposal didn't seem like going to take 3 months of working full time. Please let me know if you think otherwise.

Reply:
I think the repo lifecycle will take a while since it will require getting feedback and agreement across the project on what the different stages are. The project is also considering updating its governance and that could be part of the updating of the governance docs too.